adapters/flatads/flatadstest/exemplary/multiple-imps.json

```diff
@@ -0,0 +1,213 @@
+{
+  "mockBidRequest": {
+    "id": "test-request-id-banner",
+    "device": {
+      "ua": "test-user-agent",
+      "ip": "123.123.123.123",
+      "language": "en",
+      "dnt": 0
+    },
+    "imp": [
+      {
+        "id": "test-imp-id-banner",
+        "banner": {
+          "format": [
+            {
+              "w": 728,
+              "h": 90
+            }
+          ]
+        },
+        "ext": {
+          "bidder": {
+            "token": "66668888",
+            "publisherId": "1111"
+          }
+        }
+      },
+      {
+        "id": "test-imp-id-banner2",
+        "banner": {
+          "format": [
+            {
+              "w": 728,
+              "h": 90
+            }
+          ]
+        },
+        "ext": {
+          "bidder": {
+            "token": "66668888",
+            "publisherId": "1111"
```

Comment:
This is a good test! Optional suggestion: use different values for "token" and "publisherId" in different impressions, so expected requests uris are different.